release-please-config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "packages": {
+    ".": {}
```

Comment:
keys under "packages" are paths that are parent directories to a child package. the `"."`  path is the root path which is the only one in this non-mono-repo. see https://github.com/googleapis/release-please/blob/master/docs/manifest-releaser.md#configfile for a full config file rundown (I don't think we need anything else)